mindjunkies/forums/tests/test_views.py

```
@@ -131,8 +131,8 @@ def test_get_context_data(self):
 
         view = ForumThreadDetailsView()
         view.request = request
-        view.kwargs = {"course_slug": self.course.slug, "topic_slug": self.topic.slug}
-
+        view.kwargs = {'course_slug': self.course.slug, 'topic_id': self.topic.id}
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)